modules/luci-mod-status/htdocs/luci-static/resources/view/status/connections.js

```diff
@@ -359,23 +367,48 @@ return view.extend({
 					])
 				]),
 
-				E('div', { 'class': 'right' }, [
-					E('button', {
-						'class': 'btn cbi-button cbi-button-apply toggle-lookups',
-						'click': function(ev) {
-							if (!enableLookups) {
-								ev.currentTarget.classList.add('spinning');
-								ev.currentTarget.disabled = true;
-								enableLookups = true;
-							}
-							else {
-								ev.currentTarget.firstChild.data = _('Enable DNS lookups');
-								enableLookups = false;
-							}
+				E('br'),
 
-							this.blur();
-						}
-					}, [ enableLookups ? _('Disable DNS lookups') : _('Enable DNS lookups') ])
+				E('div', { 
+					id: 'settings-row',
+					style: 'display: flex',
+				}, [
+					E('span', { 
+						class: 'filter',
+						style: 'flex: auto',
+					}, [
+						E('input', {
+							id: 'filter-connections',
+							type: 'text',
+							placeholder: _('Type some filtering for connections...'),
```

Comment:
Perhaps revise to something more 'useful' - like an example search?

Reply:
Also please use `…` (&hellip;) instead of three ordinary dots.

Reply:
I was thinking the same, but it seemed too long to me to put examples, specially for small screens. But I can change it, maybe something short like: 
"Filter like '0.15' or 'tcp 0.15'”
I'm not English native, so any help will be appreciated...

Reply:
How about something simple like `UDP 192.168.1.1`? Then i18n is not necessary.

Reply:
The field does not have any "label" indicating what is this for, so for this reason I added the "filter" word to the placeholder. But if you think it's enough adding the sample without text, I can change it, no problem.

Reply:
Do I change this finally? If yes, what about different filters to show different samples with a separator?
`192.168.1.15 / UDP 1.15 / :443`

Maybe it's confusing...

Reply:
That’s great

Reply:
Ok, changed as discused.